okta/constants.py

```diff
@@ -47,3 +48,22 @@ def find_app_model(sign_on_mode, template_name):
         return models.SwaApplication
     # O/W SWA 3
     return models.SwaThreeFieldApplication
+
+
+OKTA_FACTOR_TYPE_TO_FACTOR = {
+    FT.CALL: models.CallUserFactor,
+    FT.EMAIL: models.EmailUserFactor,
+    FT.PUSH: models.PushUserFactor,
+    FT.QUESTION: models.SecurityQuestionUserFactor,
+    FT.SMS: models.SmsUserFactor,
+    FT.TOKEN: models.TokenUserFactor,
+    FT.TOKEN_HARDWARE: models.HardwareUserFactor,
+    FT.TOKEN_SOFTWARE_TOTP: models.TotpUserFactor,
+    FT.U_2_F: models.U2FUserFactor,
+    FT.WEB: models.WebUserFactor,
+    FT.WEBAUTHN: models.WebAuthnUserFactor
+}
```

Comment:
Are these generated, or hand coded?  what happens if we add more factor types?

Reply:
Hard-coded right now, it bugged me too.. I'll look into getting a codegen on that to future proof it 👌